api/processors/services/group_builder.py

```diff
@@ -51,7 +54,7 @@ class GroupSizeRestriction:
     def validate_size(self, size: int) -> None:
         if self.min > size:
             raise ValueError(
-                f"Group candidate must respect upper group boundaries, but {size} > {self.max}"
+                f"Group candidate must respect upper group boundaries, but {size} < {self.min}"
```

Comment:
it says "upper", so the code made sense. I remember discussing it with @paulin-mipt , but I don't remember how she managed to explain it to us why it actually made sense :) Too bad there's no comment

Reply:
I really think it was just copypaster mistake from below. Rewrote it a little bit, so it should be more clear now